src/admin/pages/newSystem.tsx

```diff
@@ -107,13 +107,15 @@ const Input: React.FC<historyProps> = props => {
 
   const post = () => {
     setSelectionCategory(selectionCategory);
+    const replace = detail.replace(/,/g, '')
```

Comment:
```suggestion
    const replacedDetail = detail.replace(/,/g, '')
```
`replace` だけよりひと目で分かる変数のが良いと思う